src/app.ts

```diff
@@ -129,6 +130,11 @@ class App {
     // LogTo webhooks
     app.post(`/account/set-default-role`, LogToWebHook.verifyHookSignature, new AccountController().setupDefaultRole)
 
+    // LogTo user info
+    app.get('/user', withLogto({ ...configLogToExpress, fetchUserInfo: true }), (req, res) => {
```

Comment:
Let's rename endpoint from `/user` to `/logto/user`

Reply:
Seems like `/user` is constant in `logto/express` framework.

Reply:
No, there is no any endpoints for `/user` used in LogTo.
Let's rename it to `/auth/user-info`